data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1617 @@
+			# Again. Wordy portion. Edit as needed.
+			choice
+				`	Walk over to where she is standing.`
+					goto walkover
+				`	Wait until she notices you.`
+			`	After several minutes of watching Taely commune with the trio of ships, she turns towards you and waves you over. "Enjoying the view?" She signs as you approach, but she continues without waiting for a response. "They are nice ships,and a major step forward in our ship development. Well worth watching." She nods to herself as she runs a hand along a fin, as if checking something.`
+				goto merganserintro
+			label walkover
+			`	Taely waves to you as you approach. "Eager to meet the latest additions to our fleet?" She pats the nose of the nearest ship. "They definitely are a beautiful addition to the Remnant patrols." She smiles to herself and runs a hand along one of the fins as if checking for something.`
+			label merganserintro
+			branch penguin
+				has "event: remnant: penguin"
+			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now, and recent breakthroughs in our materials science have enabled significant progress in our ship designs." She pauses and gestures to one. Would you like to touch them?`
+				goto touch
+			label penguin
+			`	"Captain <first>," she signs, "Please meet our first three Mergansers. I have been working on them for years now. In fact, the penguin that you helped me complete was an early prototype that was testing some of the techniques for these. With that and other recent breakthroughs in our materials science, significant progress in our ship designs has been made possible." She pauses and gestures to one. Would you like to touch them?`
+			label touch
+			choice
+				`	"No."`
+					goto turndown
+				`	"Yes."`
+			`	Taely smiles. "Just hold out your hand, and let it sense you first. Once it bumps your hand, you can touch it safely." You follow her example, and a few moments later the nearest Merganser nudges your hand. You follow Taely's lead in running your hand along it, and you discover that - despite the slimy appearance - the ship skin is dry and smooth. The surface is clearly metallic, but you can feel it stretch and twist under your hand as the ship shifts and moves. You can feel pulsing vibrations not entirely unlike the ones you are used to on the <ship>, but with subtle fluctuations.`
```

Comment:
```suggestion
			`	Taely smiles. "Just hold out your hand, and let it sense you first. Once it bumps your hand, you can touch it safely." You follow her example, and a few moments later the nearest Merganser nudges your hand. You follow Taely's lead in running your hand along it, and you discover that - despite the slimy appearance - the ship skin is dry and smooth. The surface is clearly metallic, but you can feel it stretch and twist under your hand as the ship shifts and moves. You can feel pulsing vibrations not entirely unlike the ones you are used to on the Penguin, but with subtle fluctuations.`
```
The player might not necessarily be flying the Penguin with this mission.

Reply:
No, this is intentional. It is saying that the vibrations and humms are reminiscent of whatever starship the pilot is most used to flying, but different. I'd call something like "the ship the pilot spends the most time in" but we don't track that. Next best thing is comparing to whatever the current flagship is. 

It is not specifically comparing to the Penguin.

Also, at this point in the conversation there is no